internals/cli/cmd_help.go

```diff
@@ -192,7 +192,7 @@ var HelpCategories = []HelpCategory{{
 }, {
 	Label:       "Files",
 	Description: "work with files and execute commands",
-	Commands:    []string{"ls", "mkdir", "rm", "exec"},
+	Commands:    []string{"pull", "ls", "mkdir", "rm", "exec"},
```

Comment:
Note: let's put `push` first in this list once we've merged that from master. (I think that'll be the more commonly used one.)

Reply:
The merge from master hasn't been done yet.